### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Webserver Applicationpool Edit_Screen Reader:After giving wrong input on edit field, Narrator is not reading the Error message displaying on page

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#107]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)Chrome(68.0.3440.106)
Screen Reader
MAS ID Violated: MAS 1.3.1 ,MAS 3.3.3

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Switch on narrator
4.Tab till Microsoft IIS  on the top navigation bar and press enter
5.Tab till Web server and press enter
6.Tab till Application pool and press enter
7.Tab till more command button(...) and press enter
8.Tab till Edit button and press enter
9.Tab till process tab and press enter
10.Tab till shutdown time out and give any letter or alphanumeric number
11.Check whether the Narrator is reading the error message 

Actual Result:
After giving wrong input on edit field, Narrator is not reading the Error message 

Expected Result:
Narrator should read the error suggestion displayed on page

User Impact:
Screen Reader users will not able to understand what the error message showing on page

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={e948edf0-c090-43a6-8713-66c175de7020}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
![bug_107_narrator is not reading the error message](https://user-images.githubusercontent.com/42638683/44659503-156ee800-aa22-11e8-8a31-0a5ef0c65928.png)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK controls.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Edge; #Narrator; #MAS3.3.3

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
